feature/bgp/policybase/otg_tests/import_export_multi_test/import_export_multi_test.go

```diff
@@ -415,7 +449,14 @@ func configureImportExportMultifacetMatchActionsBGPPolicy(t *testing.T, dut *ond
 		t.Fatalf("AppendNewStatement(%s) failed: %v", matchAspathSetMedStatement, err)
 	}
 
-	// TODO create as-path-set on the DUT, match-as-path-set not support.
+	// Configure my_aspath: [ "65512" ] to match_aspath_set_med statement
+	if !deviations.BgpAspathsetUnsupported(dut) {
```

Comment:
There needs to be some alternative to using aspath to achieve similar behavior.  Without this, I can't accept this deviation.

Reply:
Added the alternative with latest commit